Description: Patch is applied because autotools are not used by upstream.
 Patch setting prefix=/usr and fix install commands.
Author: Jaromír Mikeš <mira.mikes@seznam.cz>
Forwarded: no
---
 libs/Makefile |   11 +++++------
 1 file changed, 5 insertions(+), 6 deletions(-)

Index: zita-convolver/libs/Makefile
===================================================================
--- zita-convolver.orig/libs/Makefile	2011-10-28 00:26:40.000000000 +0200
+++ zita-convolver/libs/Makefile	2011-10-28 00:38:05.251285398 +0200
@@ -20,9 +20,9 @@
 
 # Modify as required.
 #
-PREFIX = /usr/local
+PREFIX ?= /usr/local
 SUFFIX := $(shell uname -m | sed -e 's/^unknown/$//' -e 's/^i.86/$//' -e 's/^x86_64/$/64/')
-LIBDIR = lib$(SUFFIX)
+LIBDIR ?= lib$(SUFFIX)
 
 
 MAJVERS = 3
@@ -31,9 +31,7 @@
 
 
 CPPFLAGS += -Wall -I. -fPIC -D_REENTRANT -D_POSIX_PTHREAD_SEMANTICS -O3
-CPPFLAGS += -ffast-math -funroll-loops 
-CPPFLAGS += -march=native
-CPPFLAGS += -DENABLE_VECTOR_MODE 
+CPPFLAGS += -ffast-math -funroll-loops
 
 LDLFAGS += 
 LDLIBS +=
@@ -56,8 +54,8 @@
 	install -d $(DESTDIR)$(PREFIX)/$(LIBDIR)
 	install -m 644 $(ZITA-CONVOLVER_H) $(DESTDIR)$(PREFIX)/include
 	install -m 755 $(ZITA-CONVOLVER_MIN) $(DESTDIR)$(PREFIX)/$(LIBDIR)
+	/sbin/ldconfig -n $(DESTDIR)$(PREFIX)/$(LIBDIR)
 	ln -sf $(ZITA-CONVOLVER_MIN) $(DESTDIR)$(PREFIX)/$(LIBDIR)/$(ZITA-CONVOLVER_SO)
-	ldconfig
 
 clean:
 	/bin/rm -f *~ *.o *.a *.d *.so.*
